roles/network_plugin/calico/templates/calico-apiserver.yml.j2

```diff
@@ -177,6 +177,9 @@ rules:
   - blockaffinities
   - caliconodestatuses
   - tiers
+  - stagedglobalnetworkpolicies      # Tigera/Calico enterprise policy staging
+  - stagedkubernetesnetworkpolicies  # Tigera/Calico enterprise policy staging
+  - stagednetworkpolicies            # Tigera/Calico enterprise policy staging
```

Comment:
IMO, this should not be within our scope of fixes. Because of the enterprise feature, it throws an error log. This should be fixed by Calico, not Kubespray.

Reply:
@morguldir @rickerc So is this an enterprise feature? I checked these CRDs and they all say Calico Cloud (except for stagednetworkpolicies which says open source).